### ISSUE TITLE
var mathjs = require('mathjs'); Error: Cannot find module 'mathjs'

### ISSUE THREAD

**Issue by millerabel** [josdejong/mathjs#512]
I've installed mathjs to Mac OS X according to the instructions on the website, using
"npm -g install mathjs", but can't load it from node.js using require('mathjs');

Here is a quick trace...

```
$ npm -v
3.4.1

$ node -v
v4.2.2

$ sudo npm -g install mathjs
Password:*
/usr/local/bin/mathjs -> /usr/local/lib/node_modules/mathjs/bin/cli.js
/usr/local/lib
└─┬ mathjs@2.4.2 
  ├── decimal.js@4.0.3 
  ├── fraction.js@3.0.0 
  ├── tiny-emitter@1.0.1 
  └── typed-function@0.10.3 

$ npm -g list mathjs
/usr/local/lib
└── mathjs@2.4.2 

$ node
> var mathjs = require('mathjs');
Error: Cannot find module 'mathjs'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at repl:1:14
    at REPLServer.defaultEval (repl.js:248:27)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.<anonymous> (repl.js:412:12)
    at emitOne (events.js:82:20)
```

**Comment by josdejong** [comment]
Does this also happen with other packages than mathjs? This sounds like a generic node related issue on your machine, not something related to mathjs. Maybe it's this issue:

http://stackoverflow.com/questions/12594541/npm-global-install-cannot-find-module

**Comment by millerabel** [comment]
It hasn't been a problem with other modules. Express has been working fine.

However, perhaps other factors in my configuration may be causing this as I recently updated Node from 0.12 to 4.4.

So I did a clean install on nodejs and npmjs...

I manually deleted NPM (```sudo rm -rf /usr/local/bin/npm /usr/local/lib/node_modules/npm```)
Reinstalled ```nodejs```, this time V5.x for Mac OS X, prebuilt binary, rather than the prebuilt 4.x.
I update ```npm``` (```sudo npm install -g npm```)
I also updated ```jshint```, ```bootlint```, and other CLI packages.

Then, reinstalled ```mathjs```, but locally. (```npm install mathjs```)

Now, ```var maths = require("mathjs");``` succeeds without throwing.

I'm going to close this issue as not reproducible with the latest nodejs/npmjs.

Thanks for your guidance!
